### PR TITLE
Update release notes for May release

### PR DESCRIPTION
**Pull request by bobsizemore**

### CODE REVIEW

**Review by dcwebermn — CHANGES_REQUESTED** [review]
For the May release notes - please include
The naming convention for environment variables has been changed from com_ibm_watson_health_* to clinical_data_annotator_*.  Any deployment that was setting com_ibm_watson_health_* environment variables should switch to the clinical_data_annotator_* naming convention.

**Review by dcwebermn — APPROVED** [review]
LGTM

File: src/pages/clouddocs/release-notes.md [hunk 1]
```diff
@@ -32,14 +40,19 @@ The following sections document the new features and changes that were included
     - Added support to allow matching CUI from the derived concept value.
     - Added support to propagate fields from the derived concept value first and use the secondary annotation as a fallback option.
     - Added support to promote dimension, frequency, and duration to the attribute if they exist in a derived concept value annotation.
-- Added warning messages when a processed ACD flow contains deprecated services.
+- Configuration Editor
+    - Added support to include linked attributes when exporting dictionary concepts.
+    - Integrated attribute search in the dictionary and UMLS concept views
+    - Context menus have been added to concept identifier and type tag fields to allow cut, paste, and delete of individual tags.
```
Comment (cagrev) [inline src/pages/clouddocs/release-notes.md:46]: for readability and to match the first two bullets, maybe:
- Added context menus to concept identifier and type tag fields to allow cut, paste, and delete of individual tags.
- Shows a warning message if a flow contains a deprecated service.

File: src/pages/clouddocs/release-notes.md [hunk 2]
```diff
@@ -16,8 +16,16 @@ toc: true
...
 The following sections document the new features and changes that were included for each release of the Annotator for Clinical Data service.
 
+## May 2023
+
+- Fixes a problem introduced in the April release that affected ACD concurrent processing.
+- Fixes an exception introduced in the April release that occurred in the Attribute Detection service processing derived concepts.
+
 ## April 2023
 
+- The naming convention for environment variables has been changed from com_ibm_watson_health_* to clinical_data_annotator_\*. Any deployment that was setting com_ibm_watson_health_* environment variables should switch to the clinical_data_annotator_* naming convention.
```
Comment (cagrev) [inline src/pages/clouddocs/release-notes.md:26]: Formatting seems to be messed up due to the asterisk, or maybe that's just the way it is presented in the review? I didn't load your branch in gatsby to check.

**Review by cagrev — APPROVED** [review]

File: src/pages/clouddocs/release-notes.md (hunk 2 above)
Comment (bobsizemore) [reply src/pages/clouddocs/release-notes.md:26]: @cagrev It just shows up that way in the review, it works fine in preview.  "*." signals markdown to italicize so I had to escape it.

**Pull request merged**